Cargo.toml

```diff
@@ -128,3 +128,10 @@ multihash = "0.19.1"
 # we import via `rust-multiaddr`.
 # This is expected to stay here until we move `libp2p-identity` to a separate repository which makes the dependency relationship more obvious.
 libp2p-identity = { path = "identity" }
+
+# lint 
+[workspace.lints.clippy]
```

Comment:
Does it work like this or do we need to use workspace inheritance in all crates to actually apply this?

Reply:
yes, lot of cargo.toml to cover 
```
[lints]
workspace = true
```
is It fine or any suggestion

Reply:
That looks like what we need, yes!